app/editor/src/features/content/list-view/components/tool-bar/filter/AdvancedSearchSection.tsx

```diff
@@ -44,7 +44,12 @@ export const AdvancedSearchSection: React.FC<IAdvancedSearchSectionProps> = () =
     const enabledOptions = seriesOptions.filter(
       (option) => option.value !== undefined && allowedSeries.has(Number(option.value)),
     );
-    return [new OptionItem('()', '()'), ...enabledOptions];
+    const sortedOptions = [...enabledOptions].sort((a, b) => {
```

Comment:
We do have a utility function for sorting that should be able to do this. `[...enabledOptions].sort(sortObject((item) => String(item.label ?? '')))`